django_celery_fulldbresult/models.py

```diff
@@ -0,0 +1,73 @@
+from __future__ import absolute_import, unicode_literals
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+from celery import states
+
+from djcelery.compat import python_2_unicode_compatible
+from djcelery.picklefield import PickledObjectField
+
+from django_celery_fulldbresult import managers
+
+
+TASK_STATE_CHOICES = zip(states.ALL_STATES, states.ALL_STATES)
+
+
+@python_2_unicode_compatible
+class TaskResultMeta(models.Model):
+    """Task result/status."""
+    task_id = models.CharField(_("task id"), max_length=255, unique=True)
+    task = models.CharField(_("task name"), max_length=200)
+    args = models.TextField(
+        _("Arguments"), blank=True, default="[]",
+        help_text=_("JSON encoded positional arguments"),
+    )
+    kwargs = models.TextField(
+        _("Keyword arguments"), blank=True, default="{}",
+        help_text=_("JSON encoded keyword arguments"),
```

Comment:
Got it! C'est bon à savoir!

Je vais peut être faire ça dans mon temps libre :p